packages/core/src/ctype/CType.schemas.ts

```diff
@@ -65,6 +122,25 @@ export const CTypeModel: JsonSchema.Schema = {
   required: ['$id', 'title', '$schema', 'properties', 'type'],
 }
 
+export const CTypeModel: JsonSchema.Schema = {
+  oneOf: [
+    CTypeModelDraft01,
```

Comment:
we're embedding the schemas here; we could also use a `$ref` here, which would however require passing the two schemas when calling `verifyObjectAgainstSchema` (`verifyObjectAgainstSchema(ctype, CTypeModel, [CTypeModelV1, CTypeModelDraft01])`)

Reply:
I like this approach more, perhaps a comment about what this actually does (i.e., not breaking existing credentials) would help!

Reply:
maybe the way to go is using the strategy described as "bundling" schemas (https://json-schema.org/understanding-json-schema/structuring.html#bundling). It's sort of a hybrid between the two options we had so far: you use `$ref`s, but you put the referenced schemas in the `definitions` section of the document so that they are still available to the validator simply by loading the bundled schema.